src/tests/unittests/validation/BindGroupValidationTests.cpp

```diff
@@ -103,3 +105,17 @@ TEST_F(BindGroupValidationTest, BufferViewOffset) {
             .GetResult();
     }
 }
+
+TEST_F(BindGroupValidationTest, BindGroupLayoutCache) {
```

Comment:
@Kangz PTAL at this new test in particular

Reply:
This won't work if/when we run unittests through the wire. LGTM for now but can you add a comment explaining this only works without wire because we get the actual pointers?

Reply:
Okay, done